.github/workflows/ci.yml

```diff
@@ -20,16 +20,16 @@ jobs:
     steps:
 
     - name: Cancel Previous Runs
-      uses: styfle/cancel-workflow-action@bb6001c4ea612bf59c3abfc4756fbceee4f870c7 # 0.10.0
+      uses: styfle/cancel-workflow-action@0.10.0 # 0.10.0
       with:
         access_token: ${{ github.token }}
 
     - name: Install Protoc
-      uses: arduino/setup-protoc@v1
```

Comment:
It is my understanding that [`v1`](https://github.com/arduino/setup-protoc/tree/v1) would actually correspond to https://github.com/arduino/setup-protoc/commit/64c0c85d18e984422218383b81c52f8b077404d3.